dev/README_RELEASE_HELM_CHART.md

```diff
@@ -273,6 +273,11 @@ cat <<EOF
 EOF
 ```
 
+```shell
+export VOTE_END_TIME=$(date --utc -d "now + 72 hours + 10 minutes" +'%Y-%m-%d %H:%M')
+export TIME_DATE_URL="to?iso=$(date --utc -d "now + 72 hours + 10 minutes" +'%Y%m%dT%H%M')&p0=136&font=cursive"
```

Comment:
The 10 minutes is to account for the time used in editing the email

Reply:
:D